kornia-py/src/lib.rs

```diff
@@ -9,82 +9,109 @@ mod pointcloud;
 mod resize;
 mod warp;
 
-use crate::icp::{PyICPConvergenceCriteria, PyICPResult};
 use crate::image::{PyImageLayout, PyImageSize, PyPixelFormat};
-use crate::io::jpegturbo::{PyImageDecoder, PyImageEncoder};
 use pyo3::prelude::*;
 
 pub fn get_version() -> String {
     let version = env!("CARGO_PKG_VERSION").to_string();
-    // cargo uses "1.0-alpha1" etc. while python uses "1.0.0a1", this is not full compatibility,
-    // but it's good enough for now
-    // see https://docs.rs/semver/1.0.9/semver/struct.Version.html#method.parse for rust spec
-    // see https://peps.python.org/pep-0440/ for python spec
-    // it seems the dot after "alpha/beta" e.g. "-alpha.1" is not necessary, hence why this works
     version.replace("-alpha", "a").replace("-beta", "b")
 }
 
+/// Helper to emit a deprecation warning in Python
+fn warn_deprecation(py: Python<'_>, message: &str) -> PyResult<()> {
+    let category = py.get_type::<pyo3::exceptions::PyDeprecationWarning>();
```

Comment:
There should be a message saying that after current version we’ll drop this . See in other functions how’s done